server_ci/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM python:3.11-slim
+
+ARG GUARDRAILS_TOKEN
+ARG GUARDRAILS_TEMPLATE="guard-template.json"
+
+# Set environment variables to avoid writing .pyc files and to unbuffer Python output
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV LOGLEVEL="DEBUG"
+ENV GUARDRAILS_LOG_LEVEL="DEBUG"
+ENV APP_ENVIRONMENT="production"
+ENV GUARDRAILS_TEMPLATE=$GUARDRAILS_TEMPLATE
+
+WORKDIR /app
+
+# Install Git and necessary dependencies
+RUN apt-get update && \
+    apt-get install -y make git curl gcc jq pipx && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN pipx install poetry
+
+# Ensure poetry is available in the PATH
+ENV PATH="/root/.local/bin:$PATH"
+
+# Copy the entrypoint script
+COPY /server_ci/entry.sh /app/entry.sh
+COPY ../ /app/guardrails
```

Comment:
if the context is project root then which thing do we want to `COPY` here?

Reply:
copy all of guardrails so i can local install it into the container and run it as the guardrails we test against